tests/bootstrap.js

```diff
@@ -85,13 +85,13 @@
     // List of scripts to load in compressed mode, instead of
     // requires.  Paths relative to root.
     compressedScripts: [
-      'blockly_compressed.js',
-      'blocks_compressed.js',
-      'dart_compressed.js',
-      'javascript_compressed.js',
-      'lua_compressed.js',
-      'php_compressed.js',
-      'python_compressed.js',
+      'build/blockly_compressed.js',
+      'build/blocks_compressed.js',
+      'build/dart_compressed.js',
+      'build/javascript_compressed.js',
+      'build/lua_compressed.js',
+      'build/php_compressed.js',
+      'build/python_compressed.js',
```

Comment:
Did you see #6218, and the reason it is still a draft PR and not merged yet?

Reply:
Yeah this was committed accidentally :/ will revert.

Reply:
https://github.com/google/blockly/pull/6292